src/pandablocks_ioc/_tables.py

```diff
@@ -291,7 +308,7 @@ def __init__(
             value,
         )
 
-        putorder_index = 0
+        putorder_index = 1
```

Comment:
Prompted by @AlexanderWells-diamond comments, we could get rid of this variable and instead do:

```python
for i, (field_name, field_record_container) in enumerate(self.table_fields_records.items()):
    ...
    field_pva_info = {
        "+type": "plain",
        "+channel": "VAL",
        "+putorder": i + 1,
        "+trigger": "",
    }
    ...
    # For the last column in the table
    if i == len(self.table_fields_records) - 1:
        ...
```